Spring-Boot/Spring-Boot-Submission/src/test/java/tw/waterball/judgegirl/springboot/submission/controllers/StudentSubmissionControllerTest.java

```diff
@@ -257,4 +259,40 @@ private SubmissionView getBestSubmission(int studentId) throws Exception {
                 .andExpect(status().isOk()), SubmissionView.class);
     }
 
+    @Test
+    void GivenAStudentHasOneSubmission_WhenRejudgeThatSubmission_ThenTheStudentShouldHaveTwoSubmissionsWithTheSameFileIds() throws Exception {
+
+        var submission = submitCodeAndGet(ADMIN_ID, ADMIN_TOKEN);
+
+        mockMvc.perform(post("/api/submissions/{submissionId}/judge", submission.getId()))
+                .andExpect(status().isOk());
+
+        allOfTheStudentSubmissionsShouldHaveTheSameFileIds(ADMIN_TOKEN);
+    }
+
+    @Test
+    void GivenTwoSubmissionsOfAQuestionFrom2Admins_WhenRejudgeAllSubmissionsOfThatQuestion_SubmissionsOfBothAdminsShouldHaveTheSameFileIds() throws Exception {
```

Comment:
rename to `GivenTwoSubmissionsOfAQuestionFrom2Admins_WhenRejudgeAllSubmissionsOfThatQuestion_Then2JudgersAreDeployedAndSubmissionsOfBothAdminsShouldHaveTheSameFileIds`